oio/common/exceptions.py

```diff
@@ -208,3 +208,19 @@ def from_response(resp, body=None):
         return cls(http_status, status, message)
     else:
         return cls(http_status, resp.reason)
+
+
+def from_response_urllib3(resp, body=None):
+    http_status = resp.status
+    cls = _http_status_map.get(http_status, ClientException)
```

Comment:
This is a duplicate of `from_response`. Just change `resp.status_code`. Add a `try` block if you worry about legacy responses to be passed to this function.